topics/icrc_25_signer_interaction_standard.md

```diff
@@ -88,28 +88,36 @@ A session must be terminated automatically after a certain period of inactivity.
 
 ## Scopes
 
-A scope is the permission to invoke specific JSON-RPC 2.0 method on the signer. A scope is identified by the `method` property which matches the `method` name of the JSON-RPC 2.0 call it relates to. The relying party requests scopes using the [`icrc25_request_permissions`](#icrc25requestpermissions) method and may revoke them using  [`icrc25_revoke_permissions`](#icrc25revokepermissions).
+A scope is the permission to invoke a specific JSON-RPC 2.0 method on the signer. A scope is identified by the `method` property which matches the `method` name of the JSON-RPC 2.0 call it relates to. The relying party requests scopes using the [`icrc25_request_permissions`](#icrc25requestpermissions) method and may revoke them using  [`icrc25_revoke_permissions`](#icrc25revokepermissions).
 
-Not all methods defined in this standard require a scope.
+None of the methods defined in this standard require a scope.
 
 ### Scope Objects
 
 Scopes are represented in JSON-RPC 2.0 messages as JSON objects with the following properties:
 - `method` (`text`): JSON-RPC 2.0 method the scope is associated with.
 
-#### Optional Properties
```

Comment:
moved to [icrc3x_canister_calls](https://github.com/dfinity/wg-identity-authentication/pull/104/files#diff-3051ebf99f07d4e1c09251345249df532674aa87c2dc61c76a5a5c1a1c7fe18cR167)